server/src/clients/mysql.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import pymysql  # type: ignore
```

Comment:
mypy hates me. I gave up - plz take a look and see if you can fix when adding the sqlalchemy models @SammyBaek

Reply:
lol ignore in import is more than fine :') what exactly is the issue?

Reply:
Mypy being a pain. We can try to clean it up sometime later.